modules/ducktests/tests/docker/run_tests.sh

```diff
@@ -158,5 +158,7 @@ if [[ -n "$MAX_PARALLEL" ]]; then
   DUCKTAPE_OPTIONS="$DUCKTAPE_OPTIONS --max-parallel $MAX_PARALLEL"
 fi
 
+"$SCRIPT_DIR"/../certs/mkcerts.sh
```

Comment:
This is mechanism for docker env. How is it going to be distributed on real nodes on Jenkins?

Reply:
first certificates are generated, then copied to the nodes